pkg/codegen/internal/test/testdata/aws-fargate.pp.py

```diff
@@ -9,15 +9,15 @@
     vpc_id=vpc.id,
     egress=[{
         "protocol": "-1",
-        "fromPort": 0,
```

Comment:
Will this be a breaking change to our Python SDKs?

Reply:
Nope--the generated program code was wrong prior to these changes. The generated schema won't change at all.